src/app/pages/account/users/user-form/user-form.component.ts

```diff
@@ -1,545 +1,352 @@
-          tooltip: helptext.user_form_home_dir_permissions_tooltip,
-        },
-      ],
-    },
-    {
-      name: helptext.user_form_auth_title_name,
-      class: helptext.user_form_auth_title_class,
-      label: true,
-      width: '50%',
-      config: [
-        {
-          type: 'textarea',
-          name: helptext.user_form_auth_sshkey_name,
-          placeholder: helptext.user_form_auth_sshkey_placeholder,
-          tooltip: helptext.user_form_auth_sshkey_tooltip,
-        },
-        {
-          type: 'select',
-          name: helptext.user_form_auth_pw_enable_name,
-          placeholder: helptext.user_form_auth_pw_enable_placeholder,
-          tooltip: helptext.user_form_auth_pw_enable_tooltip,
-          options: [
-            { label: helptext.user_form_auth_pw_enable_label_yes, value: true },
-            { label: helptext.user_form_auth_pw_enable_label_no, value: false },
-          ],
-          value: false,
-        },
-        {
-          type: 'select',
-          name: helptext.user_form_shell_name,
-          placeholder: helptext.user_form_shell_placeholder,
-          tooltip: helptext.user_form_shell_tooltip,
-          options: [],
-        },
-        {
-          type: 'checkbox',
-          name: helptext.user_form_lockuser_name,
-          placeholder: helptext.user_form_lockuser_placeholder,
-          tooltip: helptext.user_form_lockuser_tooltip,
-          isHidden: false,
-        },
-        {
-          type: 'checkbox',
-          name: helptext.user_form_sudo_name,
-          placeholder: helptext.user_form_sudo_placeholder,
-          tooltip: helptext.user_form_sudo_tooltip,
-          isHidden: false,
-        },
-        {
-          type: 'checkbox',
-          name: helptext.user_form_microsoft_name,
-          placeholder: helptext.user_form_microsoft_placeholder,
-          tooltip: helptext.user_form_microsoft_tooltip,
-        },
-        {
-          type: 'checkbox',
-          name: helptext.user_form_smb_name,
-          placeholder: helptext.user_form_smb_placeholder,
-          tooltip: helptext.user_form_smb_tooltip,
-          value: true,
-        },
-      ],
-    },
-    {
-      name: 'divider',
-      divider: true,
-    },
-  ] as FieldSet<this>[]);
+  form = this.fb.group({
+    full_name: ['', [Validators.required]],
+    username: ['', [
+      Validators.required,
+      Validators.pattern(UserService.namePattern),
+      Validators.maxLength(16),
+    ]],
+    email: ['', [Validators.email]],
+    password: ['', [Validators.required, Validators.pattern(UserService.passwordPattern)]],
+    password_conf: ['', [
+      Validators.required,
+      this.validatorsService.withMessage(
+        matchOtherValidator('password'),
+        this.translate.instant('Password and confirmation should match.'),
+      ),
+    ]],
+    password_edit: ['', [Validators.pattern(UserService.passwordPattern)]],
+    password_conf_edit: ['', [
```

Comment:
Is there a reason for another set of fields, as opposed to a conditional required validator?

Reply:
Fixed 👍🏻